rllib/agents/trainer_config.py

```diff
@@ -945,7 +941,16 @@ def offline_data(
         if actions_in_input_normalized is not None:
             self.actions_in_input_normalized = actions_in_input_normalized
         if input_evaluation is not None:
-            self.input_evaluation = input_evaluation
+            deprecation_warning(
+                old="offline_data(input_evaluation={})".format(input_evaluation),
+                new="offline_data(off_policy_estimation_methods={})".format(
+                    input_evaluation
+                ),
+                error=False,
```

Comment:
Actually, TrainerConfig objects are relatively new, so let's make this `error=True`.